Add information on updating replication_factor cloud vs open-source (#1318)
qdrant/landing_page: qdrant-landing/content/documentation/guides/distributed_deployment.md

## Replication

Qdrant allows you to replicate shards between nodes in the cluster.

Shard replication increases the reliability of the cluster by keeping several copies of a shard spread across the cluster.
This ensures the availability of the data in case of node failures, except if all replicas are lost.

### Replication factor

> When you create a collection, you can control how many shard replicas you'd like to store by changing the `replication_factor`. By default, `replication_factor` is set to "1", meaning no additional copy is maintained automatically. If you configured a default replication factor in the [Qdrant configuration](/documentation/guides/configuration/#configuration-options), this value will be the default for new collections instead. You can change that by setting the `replication_factor` when you create a collection.

Reviewer: ```suggestion
When you create a collection, you can control how many shard replicas you'd like to store by changing the `replication_factor`. By default, `replication_factor` is set to "1", meaning no additional copy is maintained automatically. The default can be changed in the [Qdrant configuration](/documentation/guides/configuration/#configuration-options). You can change that by setting the `replication_factor` when you create a collection.
```

> The `replication_factor` can also be updated for existing collections. In Qdrant open-source, this will not automatically create new replicas. See [Creating new Shard Replicas](#creating-new-shard-replicas) for a description on how to create them manually. But in Qdrant Cloud, including Hybrid Cloud and Private Cloud, the shards will be automatically replicated or removed if you change the `replication_factor`.

Reviewer: ```suggestion
The `replication_factor` can be updated for an existing collection, but the effect of this depends on how you're running Qdrant. If you're hosting the open source version of Qdrant yourself, changing the replication factor after collection creation doesn't do anything. You can manually [create](#creating-new-shard-replicas) or drop shard replicas to achieve your desired replication factor. In Qdrant Cloud (including Hybrid Cloud, Private Cloud) your shards will automatically be replicated or dropped to match your configured replication factor.
```

### Creating new shard replicas

> It is possible to create or delete replicas manually on an existing collection using the [Update collection cluster setup API](https://api.qdrant.tech/master/api-reference/distributed/update-collection-cluster). This is usually only necessary if you run Qdrant open-source. In Qdrant Cloud shard replication is handled and updated automatically.

Reviewer: ```suggestion
It is possible to create or delete replicas manually on an existing collection using the [Update collection cluster setup API](https://api.qdrant.tech/master/api-reference/distributed/update-collection-cluster). This is usually only necessary if you run Qdrant open-source. In Qdrant Cloud shard replication is handled and updated automatically, matching the configured `replication_factor`.
```
